src/predeploys.ts

```diff
@@ -0,0 +1,12 @@
+export const predeploys = {
```

Comment:
A nice to have imo would be to export these as contract objects that are already connected to the correct address

Reply:
Yeah, will leave that for a future PR.